build.gradle

```diff
@@ -1,8 +1,15 @@
+apply plugin: 'java'
+apply plugin: 'java-library'
 apply plugin: 'idea'
 apply plugin: 'eclipse'
+apply plugin: 'maven'
```

Comment:
remove unnecessary plugins and keep minimal

Reply:
removed